---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

## Experience
### Research Officer | Bioprocessing Technology Institute, A*STAR  
**2021 - Present**
  * In-silico modeling of immunotherapeutics biomanufacturing
  * Data-driven rational cell line development for optimized biologics production in CHO cell lines
  * De novo genome annotation for industry collaboration in cultured meat
  * Supervisors: Asst. Prof. Meiyappan Lakshmanan (2021-2023), Dr Zach Pang (2023-Present)

### Bioinformatics Specialist | National Cancer Centre Singapore 
**2019 - 2021**
  * Crafted and maintained specialized cloud computing (Google Cloud) data analysis pipelines
  * Analysed whole-exome data of over 1000 patients for clinical studies of various cancer predispositions
  * Supervisor: A/P Ann SG Lee

### Research Assistant | Yong Loo Lin School of Medicine, NUS 
**2018 - 2019**
  * Clinical statistics to elucidate potential cardiovascular disease risk biomarkers using metabolomic data
  * Supervised an independent student reporting on clinical applications of machine learning
  * Supervisor: Asst. Prof. Chester Drum

### Final Year Research Intern | Singapore Immunology Network, A*STAR 
**2017 - 2018**
  * Developed ezSingleCell, an R package for integrated analysis of single cell RNA-seq data ()
  * Lead maintainer for Bioconductor package cytofkit, software package for flow cytometry data analysis
  * Supervisor: Dr. Chen Jinmiao

## Education
### National University of Singapore 
**2024 - 2028 _(ongoing)_**  
Doctor of Philosophy (Ph.D.), Medicine

### AI Singapore 
**2018 - 2019**  
AI for Industry – Practical  Foundations in AI with Python

### National University of Singapore 
**2014 - 2018**  
Bachelor of Science (Honours), Major in Life Sciences

## Publications
_Most recent first_
1. Lee, N. Y., Hum, M., Zihara, S., Wang, L., <ins>Myint, M.K.</ins>, Lim, D. W. T., ... & Lee, A. S., (2023). Landscape of germline pathogenic variants in patients with dual primary breast and lung cancer. _Human genomics_, 17(1), 66. DOI: [10.1186/s40246-023-00510-7](https://doi.org/10.1186/s40246-023-00510-7)
1. Walsh, I., <ins>Myint, M.</ins>, Nguyen-Khuong, T., Ho, Y.S., Ng, S.K. and Lakshmanan, M., (2022, December). Harnessing the potential of machine learning for advancing “quality by design” in biomanufacturing. In _MAbs_ (Vol. 14, No. 1, p. 2013593). Taylor & Francis. DOI: [10.1080/19420862.2021.2013593](https://doi.org/10.1080/19420862.2021.2013593)
1. Lee, N. Y., Hum, M., Amali, A. A., Lim, W. K., Wong, M., <ins>Myint, M.K.</ins>, M. K., ... & Lee, A. S. (2022). Whole-exome sequencing of BRCA-negative breast cancer patients and case–control analyses identify variants associated with breast cancer susceptibility. _Human Genomics_, 16(1), 61. DOI: [10.1186/s40246-022-00435-7](https://doi.org/10.1186/s40246-022-00435-7)
1. Lee, N. Y., Hum, M., Ong, P. Y., <ins>Myint, M.K.</ins>, Ong, E. H., Low, K. P., ... & Lee, A. S. (2022). Germline variants associated with nasopharyngeal carcinoma predisposition identified through whole-exome sequencing. _Cancers_, 14(15), 3680. DOI: [10.3390/cancers14153680](https://doi.org/10.3390/cancers14153680)

Additional Information
------
* Project Management Fundamentals trained (16 hours)
* Technical Skills: R, Python, Bash, Snakemake, Docker, Conda, Git, Google Cloud Platform, AWS, HPC
* Certifications: GCP Big Data & Machine Learning Fundamentals